src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+.body {
+  margin: 0;
+}
+
+.star {
```

Comment:
The selector '.star' should be '.stars.star' to match the class names used in the HTML file.